src/lib/utils/delta.js

```diff
@@ -0,0 +1,49 @@
+import { isArray, isDate, isPlainObject, mapValues } from 'lodash'
+import Config from '../../config/config'
+import { on } from '../analytics/analytics'
+import logger from '../logger/js-logger'
+import { isScalar } from './object'
+
+const postMessage = data => {
+  const messenger = window.ReactNativeWebView
+
+  if ('undefined' === typeof messenger) {
+    return
+  }
+
+  messenger.postMessage(JSON.stringify(data))
+}
+
+const transformValue = value => {
+  if (isArray(value)) {
+    return value.map(transformValue)
+  }
+
+  if (isPlainObject(value)) {
+    return mapValues(value, transformValue)
+  }
+
+  if (!isScalar(value) && !isDate(value)) {
+    return
+  }
+
+  return value
+}
+
+if (Config.isDeltaApp) {
+  logger.on('*', ({ name, level, messages }) =>
```

Comment:
no need to send logs. only events.

Reply:
logs could give much more info and be more useful. if this won't be needed - I will remove it

Reply:
remove it. it will create an overload on the delta app. at the moment events are enough.